compiler/include/symbol.h

```diff
@@ -776,4 +776,12 @@ extern Symbol *gSingleVarAuxFields;
 extern std::map<FnSymbol*,int> ftableMap;
 extern std::vector<FnSymbol*> ftableVec;
 
+#define FUNC_NAME_MAX 256
+extern char llvmFuncDumpName[FUNC_NAME_MAX+1];
+extern int llvmFuncOptDump;
+
+#ifdef HAVE_LLVM
+void llvmFunctionDump(int optLevel, llvm::Function *llvmFunc = NULL, FnSymbol *chapelFunc = NULL);
```

Comment:
Let's put a comment here saying what this function does. Something like

Prints out LLVM IR for the function selected by llvmFuncDumpName at the optimization level llvmFuncOptDump. If llvmFunc and chapelFunc are NULL, this function assumes it has been called previously with non-NULL arguments for these functions.

But I find this behavior a little bit odd and wonder if it would be better to just make llvmFunctionDump's static variable in to a global. Is that what you had before? Sorry if I'm leading you in a circle..